server/wavelength/modules/economy/index.js

```diff
@@ -0,0 +1,124 @@
+// Used with nef
+
+'use strict';
+
+const FS = require(('../../../../.lib-dist/fs')).FS;
+const DB_PATH = 'config/database'; // path to your database folder
+
+module.exports = class Economy {
+	constructor(name, dbName) {
+		this.name = name;
+		this.dbName = dbName;
+		this.db = Db[dbName];
+
+		this.transactionLogs = FS('logs/transactions.txt');
+		this.tLogs = this.transactionLogs.createAppendStream();
+	}
+
+	grammarizeName(value) {
+		return value !== 1 ? `${this.name}s` : this.name;
+	}
+
+	// positive integer
+	isPInteger(value) {
+		let numValue = Number(value);
+
+		return Number.isInteger(numValue) && numValue > 0;
+	}
+
+	get(target) {
+		let tarId = toId(target);
```

Comment:
If your only using the variable once, its generally best to just use `toId(target)` in the function rather than making a variable. Exceptions for readability do exist.

Reply:
If anything, you should also assert whether or not `toId(target)` returns a valid userid string (non-empty, doesn't exceed 18 characters, et cetera). But yes, this should be moved to a calling function, and then you could just simplify the function to the following:
```js
get(userid) {
   return this.db.get(userid, 0);
}
```